### PR TITLE
Modal menu

### PR DESCRIPTION
**Pull request by LarryIVC**
### **In this PR, I did the following:**
 
- [x] Create a new branch to add a Portfolio Mobile Menu.
- [x]  Added HTML layout for the Mobile menu.
- [x]  Added styles CSS for the Mobile Menu.
- [x] Added JavaScript code for the Mobile menu. 
- [x]  Install and Test Linters in the project.

### CODE REVIEW

File: mobile.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  mainContainer.classList.remove('blurize');
+  blurContainer.classList.remove('blurize');
+}
+
+modalOpen.addEventListener('click', openModal);
+modalClose.addEventListener('click', closeModal);
+
+liPortfolio.addEventListener('click', closeModal);
+liAbout.addEventListener('click', closeModal);
+liContact.addEventListener('click', closeModal);
```
Comment (anagudelogu) [inline mobile.js]: - [x] You did an excellent job working on this Milestone! 🚀 However, regarding JavaScript best practices, this piece of code feels kind of repetitive, imagine if you want to add a new section to your portfolio in the future, you would have to: first, select the `li` element (As you're selecting them individually by id), and then, add an event listener to close the modal. 🤔 This isn't very practical as you have to repeat yourself a lot, remember the DRY rule from [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)!

To solve this, you can give all these links a common class, something like: `li-element`, then you can select all:

```js
const navigationLinks = document.querySelectorAll('.li-element')
```

This returns a NodeList with all `li-elements` and you can iterate over it and add each one of them the click listener:

```js
navigationLinks.forEach((link) => {
  link.addEventListener('click', closeModal)
}
```

This way, if you ever want to add more links, you just need to give them the `li-element` class and they will have this functionality without you needing to manually set it! Please consider applying this change 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @LarryIVC, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 
✅ Great animation! 🥇 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: mobile.js [hunk 2]
```diff
@@ -20,9 +16,11 @@ function closeModal() {
...
-modalClose.addEventListener('click', closeModal);
+const navigationLinks = document.querySelectorAll('.modal-item');
 
-liPortfolio.addEventListener('click', closeModal);
-liAbout.addEventListener('click', closeModal);
-liContact.addEventListener('click', closeModal);
\ No newline at end of file
+navigationLinks.forEach((link) => {
+  link.addEventListener('click', closeModal);
+})
```
Comment (LarryIVC) [inline mobile.js]: @anagudelogu, I implement your observation.

File: mobile.js (hunk 1 above)
Comment (LarryIVC) [reply mobile.js]: Hi @anagudelogu , I am very grateful for your observation, it helped me to learn more. 👏 

image

**Review by shella12 — APPROVED** [review]
Hi @LarryIVC and @CarlosIgreda  , 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Linters are passing ✔️ 
- Great animation effects 👍  
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**